integrations/dd-sdk-android-sqldelight/src/main/kotlin/com/datadog/android/sqldelight/SqlDelightExt.kt

```diff
@@ -30,7 +29,7 @@ fun <T : Transacter> T.transactionTraced(
     noEnclosing: Boolean = false,
     body: TransactionWithSpanAndWithoutReturn.() -> Unit
 ) {
-    withinSpan(operationName, GlobalTracer.get().activeSpan()) {
+    withinSpan(operationName, GlobalDatadogTracerHolder.get().activeSpan()) {
```

Comment:
maybe we can simply call it `GlobalDatadogTracer` instead of `GlobalDatadogTracerHolder`? wdyt?